test/test_pyfans/input.json

```diff
@@ -0,0 +1,31 @@
+{
+    "ms_filename": "../microstructures/sphere32.h5",
+    "ms_datasetname": "/sphere/32x32x32/ms",
+    "ms_L": [1.0, 1.0, 1.0],
+
+    "problem_type": "mechanical",
+    "matmodel": "LinearElasticIsotropic",
+    "material_properties":{
+        "bulk_modulus": [62.5000, 222.222],
+        "shear_modulus": [28.8462, 166.6667]
+    },
+
+    "method": "cg",
+    "error_parameters":{
+        "measure": "Linfinity",
+        "type": "absolute",
+        "tolerance": 1e-10
+    },
+    "n_it": 100,
+    "macroscale_loading":   [
```

Comment:
I would suggest to make this empty...

Reply:
only 1 entry would be sufficient

Reply:
Ah okay! Thanks, this makes it even shorter.